lib/fuse.c

```diff
@@ -163,6 +163,7 @@ struct node_lru {
 
 struct fuse_direntry {
 	struct stat stat;
+	fuse_ino_t ino;
```

Comment:
The part I don't like is

"FUSE_UNKNOWN_INO" == "enable-readdir-plus" and "0" == "disable-readdir-plus"


But I don't have a better idea yet either.

Reply:
Maybe 
```
#define FUSE_INO_NEEDS_LOOKUP FUSE_UNKNOWN_INO
```

and
```
/* disables readdir-plus in fuse-kernel for these entries */
#define FUSE_INO_NONE 0
```

?

Reply:
We don't really need to store ino in the list. We can store instead just bool plus instead so it won't look like a hack.